### ISSUE TITLE
Include broker details in metadata 

### ISSUE THREAD

**Issue by jrha** [quattor/aquilon#119]
Would this be a good idea?

In particular it would be nice to have the version of the broker written to the host plenary, but having the broker's host-name might also be useful. This would allow us to track roll-outs of changes which only take effect as the plenaries are rebuilt as well as letting us build in time-bombs for any workarounds that get upstream fixes.

`/metadata/broker/version`?
`/metadata/broker/servername`?

**Comment by stdweird** [comment]
at some point i also proposed to do this for panc by panc, but i can't remember why we choose not to. i'll try to find issue...

**Comment by ned21** [comment]
> version number

This would cause all profiles to change on every release. Since we release weekly, I would be not keen on this.

> servername

We already have this because one of our services is aqd, so the server name appears in `/system/services/aqd/...` This allows to bind hosts to e.g. a beta broker for testing new functionality with real hosts. Any reason you could not do the same?

**Comment by jrha** [comment]
> This would cause all profiles to change on every release. Since we release weekly, I would be not keen on this.

Do you flush after every release?

**Comment by ned21** [comment]
>> This would cause all profiles to change on every release. Since we release weekly, I would be not keen on this.
> Do you flush after every release?

No. I see there's an interesting angle there: if we added this but didn't flush we would be able to tell which version of the broker generated the host plenary. But then that gets confusing because it may be including plenaries generated by other versions of the broker.
